scout/server/blueprints/panels/templates/panels/gene-edit.html

```diff
@@ -44,7 +44,7 @@
             </div>
             <div class="col-md-4">
               {{ form.custom_inheritance_models.label(class="control-label") }}
-              {{ form.custom_inheritance_models(class="form-control", placeholder="free text, comma separated") }}
+              {{ form.custom_inheritance_models(class="form-control", placeholder="Valid models are: AR, AD, MT, XD, XR, X, Y") }}
```

Comment:
This placeholder should be updated. I guess sync with the constants is a bit much, but at least a one-off update, and something other than "valid" since one can actually input anything.

Reply:
Yes, right, thank you!